src/main/java/Player.java

```diff
@@ -42,7 +42,7 @@ public void takeTurn() {
 
         System.out.println("Current player : " +name);
         System.out.println("Square landed : " + newLoc.getName());
-        //landedOn(newLoc)
+        newLoc.landedOn(this);
         piece.setLocation(newLoc);
```

Comment:
you can remove setLocation. landedOn replace this method

Reply:
are you sure about this nothing sets the location of the piece in the landedOn methods (except the goToJail one of course)

Reply:
![image](https://user-images.githubusercontent.com/17824883/79442838-5b066d00-7fd9-11ea-8ab8-4e62701b9990.png)

According to the new schema, only landedOn is called

Reply:
LandedOn is abstract for square and in the spécialisations I'm not told to take care to set the location of the piece. Also in the first iteration it was the player who took care of that. So I don't really know...
![Capture1](https://user-images.githubusercontent.com/34276492/79444306-504cd780-7fdb-11ea-9b73-6238ca8fb014.PNG)
![Capture2](https://user-images.githubusercontent.com/34276492/79444311-517e0480-7fdb-11ea-94e8-ff89a657d00e.PNG)